disable_challenge.yml

```
@@ -0,0 +1,80 @@
+---
+- name: configure contestant machines to undo challenge
```

```suggestion
- name: Configure contestant machines to undo challenge
```

Als I think I would prefer one challenge runbook which can be run with a tag `on`,`off` to switch between the 2 modes. I'll take a look if this is doable with the current books.